scenes/player/Player.cs

```diff
@@ -0,0 +1,125 @@
+using Godot;
+using System;
+using Cosmobot;
+
+namespace Cosmobot
+{
+
+    public partial class Player : CharacterBody3D
+    {
+        [ExportCategory("Movement")]
+        [Export] public float WalkSpeed { get; private set; } = 5.0f;
+        [Export] public float SprintSpeed { get; private set; } = 8.0f;
+        [Export] public float Acceleration { get; private set; } = 60.0f;
+        [Export] public float AirAcceleration { get; private set; } = 15.0f;
+        [Export] public float JumpForce { get; private set; } = 4.5f;
+        [Export] public float GravityMultiplier { get; private set; } = 1.0f;
+
+        [ExportCategory("Jetpack")]
+        [Export] public float JetpackForce { get; private set; } = 3.0f;
+        [Export] public float JetpackMaxFuel { get; private set; } = 1.5f;
+        [Export] public float JetpackFuelRegenSpeed { get; private set; } = 4.0f;
+
+        public float JetpackFuel { get; private set; } = 0f;
+
+        private float _gravity = ProjectSettings.GetSetting("physics/3d/default_gravity").AsSingle();
+        private ShoulderCamera _shoulderCamera;
+
+        private bool _jetpackActive = false;
+        private bool _isSprinting = false;
+
+        private bool CanRefuelJetpack => IsOnFloor() && _jetpackActive == false;
+        private bool JetpackDisableCondition => !Input.IsActionPressed("jump_jetpack") || JetpackFuel <= 0f;
+
+        public override void _Ready()
+        {
+            _shoulderCamera = (ShoulderCamera)GetNode("ShoulderCamera");
+            _gravity *= GravityMultiplier;
+        }
+
+        public override void _Process(double delta)
+        {
+            // TEMP
+            // feature for quickly closing the game when testing
+            if (Input.IsActionJustPressed("quit_temp"))
+            {
+                GetTree().Quit();
+            }
+        }
+
+        public override void _PhysicsProcess(double delta)
+        {
+            ProcessMovement((float)delta);
+        }
+
+        #region ProcessMovement
+        private void ProcessMovement(float delta)
+        {
+            ProcessHorizontalMovement((float)delta);
+            ProcessVerticalMovement((float)delta);
```

Comment:
Redundant type casting. 
```suggestion
            ProcessHorizontalMovement(delta);
            ProcessVerticalMovement(delta);
```